### DISCUSSION TITLE
Meteorological Questions (How to create certain weather conditions)

### DISCUSSION THREAD

**Discussion by MET905** [niels747/2D-Weather-Sandbox#71] [General]
Hey weather family, I made this discussion for those that need help with creating weather conditions.
for example ask how to create freezing rain, thunderstorms ect. ASK AWAY!

**Comment by wilburcoding** [comment]
Tips for crazy storms:
- Large lakes 
- Mountains
- Maximum lake sea temperature
- Maximum lake / land evaporation
- Maximum evaporation heat
Big storms!!!!

**Comment by coolkid0281** [comment]
Tips For Snow 
I like to use the temp tool and turn the intensity on lowest and put the size to 200 and put it on Whole Width Brush and press Ctrl to cool down then turn the sea temp warmer then the air, and then you put it on Maximum lake evaporation. I also made lots of lake effect snow by doing this the wind speed got up to 40 mph in the storm!
![image](https://user-images.githubusercontent.com/110744575/225154389-17516116-b190-4a64-9985-4661d5ede737.png)

**Comment by qfmp1012** [comment]
how can I create high-elevation thunderstorms? like base at 3000 meters or more (about 9842 feet)

**Reply by Meso22** [reply]
Dry deserts near hot oceans 30c + that usually creates thunderstorms that are high based, also increased greenhouse gases
